pkg/config-v2/config.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2017 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+package config
+
+import (
+	"bytes"
+	"encoding/gob"
+	"flag"
+	"io/ioutil"
+	"os"
+	pathpkg "path"
+	"runtime"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"github.com/koding/multiconfig"
+)
+
+type Config struct {
+	OpenPitrix
+}
+
+type OpenPitrix struct {
+	Database Database
+
+	Host string `default:"127.0.0.1"`
```

Comment:
The default listening host should not be 127.0.0.1. Instead it should be 0.0.0.0. Otherwise it is not accessible from outside.

Reply:
done